src/ir/context.rs

```diff
@@ -160,6 +161,10 @@ pub struct BindgenContext<'ctx> {
     /// uses. See `ir::named` for more details. Always `Some` during the codegen
     /// phase.
     used_template_parameters: Option<HashMap<ItemId, ItemSet>>,
+
+    /// The set of `TypeKind::Comp` items found during parsing that need their
+    /// bitfield allocation units computed. Drained in `compute_bitfield_units`.
+    need_bitfield_allocation: Vec<ItemId>,
```

Comment:
So this is pretty much done so we have resolved all the types before this, right? Why is this needed? I mean, we have the bitfield offset already from clang.

If this is just for simplicity, then fine, but I'd like to understand the reasoning to defer this work.